include/examples/simple_robot.h

```diff
@@ -21,9 +26,13 @@ class SimpleRobot : virtual public kodlab::RobotBase
 public:
     int mode = 0; // Member variables encode whatever added state we need
     // Set up robot update function for state and torques
-    void Update() override
-    {
+    void Update() override {
         std::vector<float> torques(num_joints_, 0);
+        // Get states you want to use.
+        auto& imu_data = GetIMUData();
```

Comment:
I have not been consistent on this but generally everything can be const should be const.